### ISSUE TITLE
UISeeds - Button Component

### ISSUE THREAD

**Issue by emilyjablonski** [bloom-housing/bloom#3336]
Transfer the `Button` component from UIC to UISeeds. ([Audit Link](https://www.notion.so/exygy/Button-e338748b0d664546b073df5716bfd475?pvs=4)) ([Figma Docs Link](https://www.notion.so/exygy/Button-23f1ae430983449786ebbb885b7e0554?pvs=4))

**Spec notes as of May 10:**

The dual cases of Button using `<button>` or using `<a href>` under the hood will now be handled by this single Button components. The separate Link component will _not_ handle links which visually appear button-like.

**Previously:**

* Link / Button would be different components, so for the time being, let's just focus on the Button MVP flavors - the bordered / themed flavors and expanding from there

**Comment by emilyjablonski** [comment]
Waiting on tokens from @slowbot

**Comment by emilyjablonski** [comment]
Waiting on tokens from @slowbot

**Comment by slowbot** [comment]
@emilyjablonski 
Design Spec
https://www.figma.com/file/SsYt5YsgbmOPbwr5Qk2RKB/bloom-core-library?node-id=3303%3A10459&t=iAKnk6RouT6G9JBC-1

zeroheight
https://exygy.zeroheight.com/styleguide/s/100882/p/7607be-button/b/93a6d7

**Comment by slowbot** [comment]
@emilyjablonski @jaredcwhite Link Spec and Doc
Design Spec
https://www.figma.com/file/SsYt5YsgbmOPbwr5Qk2RKB/bloom-core-library?node-id=3516%3A14162&t=HUSp8iIZfcitGnqz-1

zeroheight
https://exygy.zeroheight.com/styleguide/s/100882/p/77fb4a-link/b/098a03

**Comment by emilyjablonski** [comment]
Some questions / notes!

* `Linking between a site’s pages. Use regular links instead.`
We are currently doing this - for example on the Alameda home page we have buttons that link to other pages like “View Opportunities and Resources” or “Read About How it Works”, or on the Detroit home page “See all rentals” and “Find rentals for you”.
Should we remove these instances, or is this guidance not accurate? If we are shifting are to this new pattern, are we doing this as part of uptake, and it seems like we might need new designs?

* `Style the button most users should select in a way that distinguishes it from other buttons on the page.`
What does “most users should select” mean here?

* `Avoid using generic labels like "OK" or "Submit."`
We use submit in a lot of places, is that something we are going to remove when we uptake?

* `Be mindful of the length of the button text.`
I think this needs to be more specific or at least offer generic boundaries, I wouldn’t have enough background context to know what is or isn’t too long.

Would love to add some guidance around when to use which button size.

The disabled state does not meet color contrast.

**Comment by jaredcwhite** [comment]
Closing in favor of https://github.com/bloom-housing/ui-seeds/issues/36
